apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -82,17 +95,19 @@ const providers: Provider[] = [
         throw new Error(ErrorCode.IncorrectUsernamePassword);
       }
 
-      if (user.identityProvider !== IdentityProvider.CAL) {
+      if (user.identityProvider !== IdentityProvider.CAL && !credentials.totpCode) {
```

Comment:
This is to not fail the authorization when an identity provider exists for the user, we want to let the authorization go through to check totp stuff